.github/actions/e2e-tests-contracts/action.yaml

```diff
@@ -38,7 +38,7 @@ runs:
     - name: Install Playwright Browsers (${{ inputs.browser }})
       if: inputs.browser != '' && inputs.browser != 'chromium'
       shell: bash
-      run: npx playwright install --with-deps ${{ inputs.browser }}
+      run: npx playwright install --with-deps chrome-beta chromium
```

Comment:
- Test times have decreased with the latest fixes, so we should try keeping this number at max 15 if possible

![CleanShot 2024-12-10 at 15 15 56](https://github.com/user-attachments/assets/2fe24549-6947-49cf-83f8-a350a2fbf2a2)